ion/include/ion/keyboard/layout_B2/layout_events.h

```diff
@@ -25,7 +25,7 @@ static constexpr EventData s_dataForEvent[4*Event::PageSize] = {
   U(), U(), TL(), TL(), TL(), TL(),
   T("["), T("]"), T("{"), T("}"), T("_"), T("→"),
   T("asin(\x11)"), T("acos(\x11)"), T("atan(\x11)"), T("="), T("<"), T(">"),
-  TL(), TL(), TL(), T("()"), U(), U(),
+  TL(), TL(), TL("Ω"), T("()"), U(), U(),
```

Comment:
Why use a TextLess fonction ? it can't work you need a text function so T("\...") ...